docs/dedukti.rst

```diff
@@ -1,26 +1,22 @@
 Compatibility with Dedukti
 ==========================
 
-One of the aims of Lambdapi is to remain compatible with
-`Dedukti <https://deducteam.github.io/>`__. Currently, a second parser
-is included in Lambdapi in order to support the legacy syntax of
-Dedukti. The grammar supported by that parser can be seen in
-:download:`syntax_dedukti.bnf`.
-It should be extended whenever the syntax of Dedukti is
-extended, but such changes should be discouraged.
+Lambdapi can also read most of
+`Dedukti <https://deducteam.github.io/>`__ files (extension ``.dk``). The
+Dedukti grammar currently supported by Lambdapi can be seen in
+`syntax_dedukti.bnf <https://raw.githubusercontent.com/Deducteam/lambdapi/master/docs/syntax_dedukti.bnf>`__.
+It should be extended whenever the syntax of Dedukti is extended.
 
-Note that files in the legacy (Dedukti) syntax are interoperable with
-files in the Lambdapi syntax. The correct parser is selected according
-to the extension of files. The extension ``.dk`` is preserved for legacy
-files, and the extension ``.lp`` is used for files in the Lambdapi
-syntax.
+For instance, Lambdapi is able to type-check all the generated
+libraries that were aimed at Dedukti (with minor
+modifications). Translation scripts are provided in the ``libraries``
+folder.
```

Comment:
This description is a bit misleading isn't it? The `libraries` folder allows to type check libraries that were designed for Dedukti, it does not translate.